File: src/main/java/cardgame/Dealer.java
package cardgame;

import java.util.Random;

public class Dealer {
	private final Random random;

	public Dealer() {
		this.random = new Random();
	}

	public void deal(Player player, int cardsToDraw) {
		player.setCards(null);
		char[] cards = new char[cardsToDraw];
		for (int i = 0; i < cards.length; i++) {
			int randomPick = random.nextInt(100);
			cards[i] = CardDeck.get(randomPick);
		}
		player.setCards(cards);
	}
}

File: dealer.js
const CardDeck = require('./carddeck');

module.exports = function (player, cardsToDraw
) {
    player.setCards(null);
    let cards = [];
    for (let i = 0; i < cardsToDraw; i++) {
        let randomPick = Math.random(100) * 100;
        cards.push(CardDeck.get(randomPick));
    }
    player.setCards(cards);
}